tzsp2pcap.c

```diff
@@ -438,6 +439,12 @@ int main(int argc, char **argv) {
 			break;
 		}
 
+    case 'a':
+      if (my_pcap.rotation_interval) {
+			  my_pcap.rotation_start_time = (int)(my_pcap.rotation_start_time / my_pcap.rotation_interval) * my_pcap.rotation_interval;
```

Comment:
This truncation needs to occur after argument parsing.

Arguments are processed in order, and the effects of `-a` depend on the effects of `-G`. As is, if the arguments are `-a -G $n` then the `-a` option will have no effect.